Explore an HCA Data Set in Scanpy (Nov 2018)/README.md

```diff
@@ -16,6 +16,11 @@ distribution. For some users, it may be necessary to use `pip3` instead.
 pip install -r ./REQUIREMENTS.txt
 ```
 
+Depending on your platform, you might need to install additional depenencies:
+
+    $ sudo apt-get install libblas-dev liblapack-dev libatlas-base-dev gfortran libhdf5-dev
```

Comment:
What should users with non-Debian distros (e.g., Mac homebrew users or Fedora yum users) do?

Reply:
Punting this to next sprint per discussion